yarpl/include/yarpl/flowable/FlowableOperator.h

```diff
@@ -246,6 +246,58 @@ class TakeOperator : public FlowableOperator<T, T> {
 };
 
 template <typename T>
+class SkipOperator : public FlowableOperator<T, T> {
+ public:
+  SkipOperator(Reference<Flowable<T>> upstream, int64_t offset)
+      : FlowableOperator<T, T>(std::move(upstream)), offset_(offset) {}
+
+  void subscribe(Reference<Subscriber<T>> subscriber) override {
+    FlowableOperator<T, T>::upstream_->subscribe(
+        Reference<Subscription>(new Subscription(
```

Comment:
`make_ref<Subscription>(Reference<Flowable<T>>(this), offset_, std::move(subscriber))`